### PR TITLE
LPS-168728 Added some style fixes and fix testflow taskId is undefined

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/index.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const Sidebar = () => {
 			className: 'mt-3',
 			element: (
 				<div
-					className={classNames('pt-3', {
+					className={classNames(' testray-sidebar-item ', {
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/index.tsx:72]: Remove unnecessary space in (' ')

```suggestion
					className={classNames('testray-sidebar-item', {
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx [hunk 2]
```diff
@@ -64,85 +65,145 @@ const SubtaskCard: React.FC<SubtaskCardProps> = ({expanded, subtask}) => (
...
-							score: subtask.score,
-						}}
-					/>
-				</div>
-			))}
-
-			{tasks.map((taskUser: TestrayTaskUser, index) => (
-				<div className="mb-5 mt-6" key={index}>
-					{expanded && (
+			{!!tasks.length && (
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx:87]: Remove this condition

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx [hunk 3]
```diff
@@ -64,85 +65,145 @@ const SubtaskCard: React.FC<SubtaskCardProps> = ({expanded, subtask}) => (
...
+		subTasks,
+		tasks,
+	} = useSidebarTask();
 
 	const justifyContentSidebar = {
 		'justify-content-between': expanded,
 		'justify-content-center': !expanded,
 	};
+
 	if (!tasks) {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx:81]:
```suggestion
	if (!tasks.length) {
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx [hunk 4]
```diff
@@ -64,85 +65,145 @@ const SubtaskCard: React.FC<SubtaskCardProps> = ({expanded, subtask}) => (
...
+								'd-flex mb-2',
+								justifyContentSidebar
+							)}
 						>
-							<span>{taskUser?.task?.name}</span>
+							<TaskBadge
+								className={classNames({
+									'mr-3': expanded,
+								})}
+								count={subTasks.length as number}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx:118]:
```suggestion
								count={subTasks.length}
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx [hunk 5]
```diff
@@ -64,85 +65,145 @@ const SubtaskCard: React.FC<SubtaskCardProps> = ({expanded, subtask}) => (
...
+						className={classNames(
+							'align-items-center d-flex task-sidebar-title',
+							justifyContentSidebar
+						)}
+					>
+						<span
+							className={classNames('font-weight-bold', {
+								'sidebar-item-hidden': !expanded,
+								'sidebar-item-visible': expanded,
+							})}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx:99]: This 

```
{
 								'sidebar-item-hidden': !expanded,
 								'sidebar-item-visible': expanded,
 							}
```

Is used here and in line: 124, you could create a variable and use both places

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/index.tsx (hunk 1 above)
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/index.tsx:72]: Space

```suggestion
					className={classNames('testray-sidebar-item ', {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**